testsuite/tests/jst-modular-extensions/user_error1.ml

```diff
@@ -0,0 +1,27 @@
+(* TEST
+   ocamlc_byte_exit_status = "2"
+   * setup-ocamlc.byte-build-env
+   ** ocamlc.byte
+   *** check-ocamlc.byte-output
+*)
+
+(* What happens if the user tries to write one of the ocaml-jst extensions in
+   terms of extension nodes but messes up?  In practice we don't expect to ever
+   see these errors, but one never knows (and a bug in our desugaring could
+   cause them).  The let-binding is named after the constructor in
+   [extensions.ml] representing this particular error. *)
+
+(* We cannot use an expect-test here, because these are essentially parsing
+   errors. The expect-test infrastructure uses Ast_mapper to prepare its
+   input, and the call to Ast_mapper fails with a bogus extension setup,
+   because it tries to decode extensions. *)
```

Comment:
Why split these files up?  They all start with the same stanza, too.  Was it technically necessary?  Do you find it clearer?